doc/design/gan_api.md

```diff
@@ -0,0 +1,134 @@
+'''
+GAN implementation, just a demo.
+'''
+# pd for short, should be more concise.
+from paddle.v2 as pd
```

Comment:
use
\```python
\```

that will give a python syntax highlight.

Reply:
Done.